Conjugar/ConjugationCell.swift

```diff
@@ -50,6 +50,7 @@ class ConjugationCell: UITableViewCell {
         conjugation = personNumber.pronoun + " " + conjugation
       }
       self.conjugation.attributedText = conjugation.conjugatedString
+        self.conjugation.setAccessibilityLabelInSpanish(conjugation)
```

Comment:
This is indented too far.

Reply:
Fixed.

By the way, you could consider setting the project's indent and tab settings specifically. If I'm correct, except in actual overrides, contributors would adopt that automatically in Xcode, to avoid these kinds of issue.

Reply:
Do you mean specify them in `contributing.md`? Is there another way?

Reply:
If you select the project in Xcode, you have some settings for it in the right sidebar.

<img width="274" alt="Screen Shot 2020-10-21 at 09 11 30" src="https://user-images.githubusercontent.com/904596/96685538-8a79a600-137d-11eb-9052-0d70e069d992.png">

<img width="259" alt="Screen Shot 2020-10-21 at 09 11 39" src="https://user-images.githubusercontent.com/904596/96685553-8f3e5a00-137d-11eb-80ad-89e284ae0fec.png">